lib/build.ts

```diff
@@ -31,6 +31,9 @@ function getMajor(nodeVersion: string) {
 function getConfigureArgs(major: number, targetPlatform: string): string[] {
   const args: string[] = [];
 
+  // Use Ninja instead of GNU make
+  args.push('--ninja');
```

Comment:
I think this should be wrapped in a guard if we are on windows so the arg doesn't propagate to windows builds.
e.g.
```
  // Use Ninja instead of GNU make
  if (hostPlatform !== 'win') {
    args.push('--ninja');
  }
```